Client/Frontend/Browser/Tab.swift

```diff
@@ -256,6 +256,31 @@ class Tab: NSObject {
         contentScriptManager.helpers.removeAll()
     }
     
+    func clearHistory(config: WKWebViewConfiguration) {
+        guard let webView = webView,
+              let tabID = id  else {
+            return
+        }
+        
+        // Remove the tab history from saved tabs
+        TabMO.removeHistory(with: tabID)
+
+        let allBase64Decoded = Data(base64Encoded: "QWxs", options: Data.Base64DecodingOptions(rawValue: 0))
+            .map({ String(data: $0, encoding: .utf8) }) ?? "" // "All" text in base64Decoded
+              
+        // Clear backforward list
+        let argument: [Any] = ["_", "r", "\u{65}", "mov", "\u{65}", // _remove
+                             "\(allBase64Decoded ?? "")", // All
+                             "It", "\u{65}", "ms"] // Items
+
+        let method = argument.compactMap { $0 as? String }.joined()
+        let selector: Selector = NSSelectorFromString(method)
```

Comment:
I think this is still over the top a bit, also please add some comments around what private API we're using and why. Otherwise seems like this works, tested it and the UI reacts appropriately showing no back/forward options and restoring without the previous history either.

Reply:
Unnecessary obfuscation is removed and comments are added